Chatkit/Networking/Client/PCBaseClient+Extensions.swift

```diff
@@ -0,0 +1,26 @@
+import Foundation
+import PusherPlatform
+
+internal extension PPBaseClient {
+    
+    // MARK: - Properties
+    
+    private static let hostname = "pusherplatform.io"
+    private static let hostnameSeparator = "."
+    private static let instanceLocatorSeparator: Character = ":"
+    
+    // MARK: - Internal methods
+    
+    class func host(for instanceLocator: String) throws -> String {
+        let components = instanceLocator.split(separator: PPBaseClient.instanceLocatorSeparator)
+        
+        guard components.count == 3 else {
+            throw NetworkingError.invalidInstanceLocator
+        }
+        
+        let cluster = components[1]
```

Comment:
I was thinking if we could use `enum` to describe components instead of numbers but it might be overkill.

Reply:
Well, we can do that or we can improve Platform SDK to handle this value in a more structured way.